extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzRuntimeConfig.java

```diff
@@ -25,6 +25,12 @@
     @ConfigItem(defaultValue = "5")
     public int threadPriority;
 
+    /**
+     * The threshold  for how long a trigger should be misfired (ms).
+     */
+    @ConfigItem(defaultValue = "60000")
+    public long misfireThreshold;
```

Comment:
```suggestion
    /**
     * Defines how late the schedulers should be to be considered misfired..
     */
    @ConfigItem(defaultValue = "60")
    public Duration misfireThreshold;
```
let's make it a duration type.